tests/everything.rs

```diff
@@ -143,7 +143,7 @@ fn test_rustfix_with_file<P: AsRef<Path>>(file: P) -> Result<(), Box<Error>> {
 
     let mut fixed = code.clone();
 
-    for sug in suggestions {
+    for sug in suggestions.into_iter().rev() {
```

Comment:
needs a test

Reply:
Only possible with an additional test that has a replacement that condenses n lines into m<n  lines followed by ≥1 replacements.

…which I just added